CI/physmon/workflows/physmon_vertexing.py

```diff
@@ -60,10 +60,13 @@ def run_vertexing(fitter, mu, events):
             s,
             MomentumConfig(1.0 * u.GeV, 10.0 * u.GeV, transverse=True),
             EtaConfig(-3.0, 3.0),
+            PhiConfig(0.0, 360.0 * u.degree),
```

Comment:
This is not the default?

Reply:
it is but I decided not to trust the defaults :smile: